### DISCUSSION TITLE
Well Defined Issues?

### DISCUSSION THREAD

**Discussion by rakshit087** [ultrasoundmoney/frontend#233] [Ideas]
I want to contribute but as of now the issues are very abstract and vague, it would be awesome if there were more well defined issues.
Anyways awesome dashboard, keep up the good work :3

**Comment by alextes** [comment]
Sorry for the slow reply, I'll take a look at this when there's more capacity. We have tons of issues defined in our backlog actually. I still need to figure out a way we can have people meaningfully contribute.

Thanks for the kind words 🤗 !
